examples/apigee-x-vpn-hub-and-spoke-design/dns_peering.tf

```
@@ -0,0 +1,39 @@
+resource "google_dns_managed_zone" "apigee_x_project_forwarding_zone" {
+  project     = google_project.apigee_x_project.project_id
+  name        = "apigee-x-project-forwarding-zone"
+  dns_name    = "${var.private_zone_domain}"
```

Interpolation is not necessary here and throughtout
```suggestion
  dns_name    = var.private_zone_domain
```